components/ILIAS/User/classes/Settings/class.ilPersonalSettingsGUI.php

```diff
@@ -784,7 +784,7 @@ protected function deleteOwnAccountStep3(): void
             $mmail->Send();
         }
 
-        $this->log->root()->log('Account deleted: ' . $ilUser->getLogin() . ' (' . $ilUser->getId() . ')');
+        $this->log->root()->log('Account deleted: ' . $this->user->getLogin() . ' (' . $this->user->getId() . ')');
```

Comment:
This commit does look like it does not belong to this PR

Reply:
You are right